membership-attribute-service/app/wiring/AppLoader.scala

```diff
@@ -16,28 +19,24 @@ import play.api.mvc.EssentialFilter
 import play.filters.cors.{CORSConfig, CORSFilter}
 import play.filters.csrf.CSRFComponents
 import router.Routes
-import services.mail.{EmailData, QueueName, SendEmail, SendEmailToSQS}
+import services.mail.{QueueName, SendEmail, SendEmailToSQS}
 import services.salesforce.ContactRepository
 import services.stripe.{BasicStripeService, ChooseStripe}
 import services.subscription.SubscriptionService
 import services.zuora.rest.ZuoraRestService
 import services.zuora.soap.ZuoraSoapService
-import services.{
-  CatalogService,
-  ContributionsStoreDatabaseService,
-  HealthCheckableService,
-  MobileSubscriptionServiceImpl,
-  PostgresDatabaseService,
-  SupporterProductDataService,
-}
+import services._
 
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext
 
 class AppLoader extends ApplicationLoader {
   def load(context: Context): Application = {
     LoggerConfigurator(context.environment.classLoader).foreach {
       _.configure(context.environment)
     }
+    val loggerPackageName = classOf[SafeLoggerImpl].getPackageName
+    LoggerFactory.getILoggerFactory.asInstanceOf[LoggerContext].getFrameworkPackages.add(loggerPackageName)
```

Comment:
this is needed so that the logger doesn't add `SafeLogger.scala:49` into the logs, instead of the real file/line number

Reply:
Nice! You could leave this comment in the code?

Reply:
good idea
```suggestion
    val loggerPackageName = classOf[SafeLoggerImpl].getPackageName
    // at the moment we get SafeLogger.scala:49 type things in the logs
    // adding to the FrameworkPackages makes logback skip over and find the real line of code
    LoggerFactory.getILoggerFactory.asInstanceOf[LoggerContext].getFrameworkPackages.add(loggerPackageName)
```